### PR TITLE
Create privacypolicy.md

### PR DESCRIPTION
**Pull request by tlubowe**
new subpage for Privacy policy (hope this formatting is right?)

### CODE REVIEW

**Comment by markf94** [comment]
Thanks @tlubowe! It seems like you added two files: `privacy_policy.md` and `privacypolicy.md`. I'm assuming this might be an accident? If yes, please just delete the one you don't want to link to.

File: qosf.org/privacy_policy.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Collection of Personal Information
+
+We collect information that you voluntarily provide to us such as your name, mailing address or email. 
+
+## The Ways QOSF Uses Information
+
+Before forwarding us any personal information, please be advised that any information gathered on our Sites may be used in the aggregate for research and development relating to our Sites and/or for future site or program development and, if you ask us to, to send you promotional materials. In particular, we may use information gathered about you for the following purposes: to monitor interest in our range of programss and to assist us to tailor the content of our Sites to your needs by collecting information about your preferences; to create a profile relating to you in order to show you the content that might be of interest to you and to display the content according to your preferences; to protect, investigate and deter against fraudulent, unauthorized or illegal activity; and, in circumstances where you have indicated that you wish to receive additional information, to send you information about us and informational material about our programs.
```
Comment (markf94) [inline qosf.org/privacy_policy.md:25]: `programss` >> `programs`
```suggestion
Before forwarding us any personal information, please be advised that any information gathered on our Sites may be used in the aggregate for research and development relating to our Sites and/or for future site or program development and, if you ask us to, to send you promotional materials. In particular, we may use information gathered about you for the following purposes: to monitor interest in our range of programs and to assist us to tailor the content of our Sites to your needs by collecting information about your preferences; to create a profile relating to you in order to show you the content that might be of interest to you and to display the content according to your preferences; to protect, investigate and deter against fraudulent, unauthorized or illegal activity; and, in circumstances where you have indicated that you wish to receive additional information, to send you information about us and informational material about our programs.
```

**Comment by mstechly** [comment]
@markf94 oh yes, I've renamed it while making a PR to be more consistent with other files, but something either went wrong or @tlubowe added it again 🤷 
@tlubowe please fix that to avoid further complications ;)

**Comment by mstechly** [comment]
@markf94 regarding this comment:
https://github.com/qosf/qosf.org/pull/47#issuecomment-771219030
Actually, do you know how to add a note with privacy policy to every single subpage we have? I tried to do that yesterday but failed :(

**Comment by markf94** [comment]
@mstechly yep I do know that. In that case, we have to modify the template HTML file. Let's just merge this MR and then I can add that!

**Comment by tlubowe** [comment]
yep definitely not intentional. We can keep privacy_policy.md I am pretty sure I deleted the other one.

**Review by tlubowe — COMMENTED** [review]
I think this is good to go

**Review by tlubowe — COMMENTED** [review]
Hope I got it this time - sorry if my git skillz are garbage

**Comment by mstechly** [comment]
Let's merge @markf94 ?

**Pull request merged**